### PR TITLE
Squidstyle

### PR DESCRIPTION
**Pull request by AsciiSquid**
## About The Pull Request

Adds new customization options for squid people in the form of three new face styles.

## Why It's Good For The Game

Squid Gang Squid Gang

## Changelog
:cl:
add: Added squid face customization
/:cl:

### CODE REVIEW

**Comment by hackerjsimon** [comment]
yes also give martial arts

**Review by ike709 — APPROVED** [review]

**Review by eLeCtrOssSnake — APPROVED** [review]

**Comment by ike709** [comment]
Conflicts.

**Review by MarkSuckerberg — CHANGES_REQUESTED** [review]
Fix the conflicts, then it'll be fine.

File: code/modules/mob/dead/new_player/sprite_accessories.dm [hunk 1]
```diff
@@ -2089,3 +2089,23 @@
...
+
+/datum/sprite_accessory/squid_face/
+	icon = 'icons/mob/mutant_bodyparts.dmi'
+	color_src = MUTCOLORS
+
+/datum/sprite_accessory/squid_face/squidward
+	name = "Squidward"
+	icon_state = "squidward"
+
+/datum/sprite_accessory/squid_face/ilithid
```
Comment (ATH1909) [inline code/modules/mob/dead/new_player/sprite_accessories.dm:2101]: "illithid" is spelled with two "l"s, not one "l"
(don't forget to update the other mentionings of the word "illithid"!)

**Comment by ike709** [comment]
I don't expect this to be finished but I'll give it the benefit of the doubt and just mark it as stale for now.

**Comment by ike709** [comment]
Stale.

**Pull request closed without merging**